src/layers/EarthEngine.js

```diff
@@ -18,6 +18,7 @@ class EarthEngine extends Layer {
     addTo = map =>
         new Promise((resolve, reject) => {
             this._map = map
+            this._isLoading = true
```

Comment:
Would it make sense to set _isLoading = false in case the promise rejects ?

Reply:
This also makes sense to me. Even if loading was unsuccessful, it is still completed.

Reply:
I'm also wondering if setting _isLoading = true should be after line 23? @turban

Reply:
yes, you can move it to after `if (map.styleIsLoaded()) {`